packages/instui-cli/lib/commands/create.js

```diff
@@ -47,6 +47,7 @@ exports.builder = (yargs) => {
     })
   }
 
+  // TODO: remove create app script and any related code
```

Comment:
We should create a ticket about this. IMO it's not a breaking change, we told people that we only have the "no breaking change without major version release" policy only for UI code, not for configuration or build scripts.

Reply:
INSTUI-3580